Logger/BaseLoggerMixins.cs

```diff
@@ -1,7 +1,58 @@
-﻿namespace Logger
+﻿using System;
+
+namespace Logger
 {
     public static class BaseLoggerMixins
     {
-        
+        public static void Error(this BaseLogger baseLogger, string message, params object[] args)
+        {
+            if (baseLogger is null || message is null || message == "")
```

Comment:
There is a handy helper method for checking if a string is null or empty.
```suggestion
            if (baseLogger is null || string.IsNullOrEmpty(message))
```